pkg/controller/machinepool/machinepool_controller.go

```diff
@@ -309,6 +315,85 @@ func (r *ReconcileRemoteMachineSet) Reconcile(ctx context.Context, request recon
 		return reconcile.Result{}, err
 	}
 
+	// Reconcile local CAPI MachineSets when cd.Spec.MachineManagement.Central enabled
+	if cd.Spec.Platform.AWS != nil && cd.Spec.MachineManagement != nil && cd.Spec.MachineManagement.Central != nil {
+		logger.Info("reconciling local machinesets")
+		return r.reconcileLocalMachineSets(pool, cd, masterMachine, logger)
+	}
+
+	// Reconcile remote MachineSets
+	logger.Info("reconciling remote machinesets")
+	return r.reconcileRemoteMachineSets(pool, cd, masterMachine, remoteClusterAPIClient, logger)
+}
+
+// reconcileAWSMachineTemplates ensures AWSMachineTemplates are configured for local CAPI MachineSets
```

Comment:
it seems odd to have AWS function in the general machinepool_controller instead of the aws actuator?